refining content for core-data

meganmason opened the pull request:
added content to 03_database_preview; 2nd and 3rd examples are nearly complete. Need to finish the geojson part, and recap. Images would help too!

I will add content to notebook 02, and the final map figure in notebook 01

Commit:
added content to 03_database_preview; 2nd and 3rd example are nearly complete. Need to finish the geojson part, and recap. Images would help too!

Commit:
fixed two spelling errors in third notebook

Commit:
updates to notebook 1; worked on the map from the airplane

Commit:
continued updates on notebook 1

Commit:
updating coredata branch with a git merge main

Commit:
fixed typos in first notebook

JessicaS11 commented:
@meganmason QC picked up a few spelling errors and tagged them. Other than fixing those, are you planning to add more to these tutorials in this PR (i.e. is it ready for review)?

Commit:
notebook 1 is now done. The rest of 2 will be quick, and 3 just needs to be checked after the database rebuild and adjusted for the TS example.

Commit:
commiting after a git merge main

Commit:
fixed 2 typos and corrected a figure caption number.

micahjohnson150 commented:
@meganmason yesterday @scottyhq was having issues with a reference to `ImageData.surveyors` in your notebook. So to move forward he temporarily disabled one of your notebooks. Looks like you fixed the reference to surveyors here but I think it wont render without removing it from the [_config.yml ](https://github.com/snowex-hackweek/website2022/blob/45ad291cbb074503d048bf7f5422de177415ce70/book/_config.yml#L43)

Commit:
second notebook is complete. Doing a few touch ups on notebook 3

Commit:
notebook 3 is complete. Fixed 3 typos in notebook 2. One is not a typo, it's a site name.

meganmason commented:
okay, team! This is ready. There might be some requested changes, a typo, or a check that doesn't pass, but I'm finished with the content. I'll watch the checks and see which ones I can manage.

Commit:
fixed a typo in notebook3

meganmason committed:
Update _config.yml

Removed temp. disabled notebook book from the _config.yml after correcting ImageData.surveyors to ImageData.observers.

meganmason commented:
thanks @micahjohnson150 for the tip on the _config.yml file. I removed it and pushed anew.

meganmason commented:
How does one handle a supposed spell check error that is not an error? In my case it's a table with a site name "BA flux tower" and the request is to change 'BA' to 'BY' or 'BE', which is not correct.

jomey commented:
> How does one handle a supposed spell check error that is not an error? In my case it's a table with a site name "BA flux tower" and the request is to change 'BA' to 'BY' or 'BE', which is not correct.

Add the “BA” at the end of this list:
https://github.com/snowex-hackweek/website2022/blob/e4c2652549353647689fbc22aedc77706e95ee88/.github/workflows/qaqc.yaml#L36

meganmason committed:
Update qaqc.yaml

added to the ignore_words_list

meganmason commented:
thanks @jomey ! I changed the file on my branch and pushed it up. Could you remind me how to "deploy" the preview?

scottyhq committed:
lowercase ba

scottyhq commented on book/tutorials/core-datasets/01_data_coverage.ipynb:
```diff
@@ -48,37 +75,63 @@
+    "These are two different sampling strategies, but result in the same types of measurements and data set products. Depending on the  research application it may not matter at all which you choose to work with, or even combine! The important thing to grasp is the difference in spatial and temporal extent of the campaign periods. If the sampling protocols or data set products change over time, i ...
+    "\n",
+    "<figure>\n",
+    "<img src='content/01_campaign_venn_diagram.jpg'>\n",
```
```suggestion
    "<img src='./content/01_campaign_venn_diagram.jpg'>\n",
```
i think you need relative paths

scottyhq reviewed (changes requested):
@meganmason the preview is automatically updated with each commit, so your content is here https://deploy-preview-73--snowex2022.netlify.app/tutorials/core-datasets/index.html . The images are not rendering currently, I think the paths need to be relative to the notebook (see comment)

meganmason committed:
Update book/tutorials/core-datasets/01_data_coverage.ipynb

Co-authored-by: Scott Henderson <scottyhq@gmail.com>

Commit:
fixed paths to images - they needed to be relative to the notebook

Commit:
committing the change to qaqc.yaml

meganmason commented:
made all images have relative file paths, but the preview still doesn't seem to be rendering them?

meganmason commented:
The pattern appears to be all images I have a figure capture for are NOT rendering, the few images without figure captions are okay.

Commit:
testing a different figure and figure caption syntax in hopes that it will render

Commit:
changed figure/figure caption syntax in notebook 1, checking to see if they render before doing the others.

Commit:
fixed figures/figure capions on notebook 2 and 3

Commit:
forgot to clear the outputs, clean notebook now

meganmason commented:
Images all render and required checks pass. I think we're good to merge by the AM
